data/sles4sap/qe_sap_deployment/qesap_azure.yaml

```diff
@@ -1,35 +1,40 @@
-provider: "%PROVIDER%"
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Generic yaml template for use with qe-sap-deployment project: https://github.com/SUSE/qe-sap-deployment
+provider: 'azure'
 apiver: 3
 terraform:
   variables:
-    az_region: "%REGION%"
-    deployment_name: "%DEPLOYMENTNAME%"
-    os_image: "%OS_VER%"
-    public_key: "%SSH_KEY_PUB%"
-    hana_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
-    iscsi_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
-    hana_instancetype: "%HANA_INSTANCE_TYPE%"
-    vnet_address_range: "%VNET_ADDRESS_RANGE%"
-    subnet_address_range: "%SUBNET_ADDRESS_RANGE%"
-
+    az_region: '%REGION%'
+    deployment_name: '%DEPLOYMENTNAME%'
+    os_image: '%OS_VER%'
+    public_key: '%SSH_KEY_PUB%'
+    hana_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    iscsi_remote_python: '%ANSIBLE_REMOTE_PYTHON%'
+    hana_instancetype: '%HANA_INSTANCE_TYPE%'
```

Comment:
`qe-sap-deployment/terraform/azure` does not have variable like `hana_instancetype`. It is an AWS only variable.